views/show.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    {{#angryBirdsList}}
```

Comment:
`{{#angryBirdsList}}` we use this syntax when we need to iterate through an array but we dont have any, we just need to show on bird 
```
<h1> Name: {{angryBirdsList.name}} </h1>
<h2> Abilities: {{angryBirdsList.abilities}} </h2>
<h2> Group: {{angryBirdsList.groub}} </h2>
<img src="{{angryBirdsList.image}}" alt="{{angryBirdsList.name}}">
<h3> <a href="/angry_birds"> Back to angry birds list</a></h3>
<h3>  <a href+"/"> Back To Homepeage </a></h3>
```